libs/ballot-interpreter/src/hmpb-ts/types.ts

```diff
@@ -130,6 +133,19 @@ export interface ScoredPositionArea {
   score: UnitIntervalScore;
 }
 
+/**
+ * Conversion intermediary for marks containing the grid position, associated
+ * contest option, the mark score, and the write-in area score if applicable.
+ */
+export interface ScoredContestOption {
+  option: ContestOption;
+  gridPosition: GridPosition;
+  scoredMark: ScoredBubbleMark;
+  markStatus: MarkStatus;
+  scoredWriteInArea?: ScoredPositionArea;
+  writeInAreaStatus: WriteInAreaStatus;
+}
```

Comment:
Should this type just live in `legacy_adapter.ts` since it's only used there? I think this file is generally meant for types that interface between TS and Rust